packages/decap-cms-core/src/lib/stega.ts

```diff
@@ -1,6 +1,5 @@
 import { vercelStegaEncode } from '@vercel/stega';
-
-import { isImmutableMap, isImmutableList } from './types';
+import { isImmutableMap, isImmutableList } from 'decap-cms-lib-util/src/types';
```

Comment:
Looking at this again, I _think_ this will actually have a similar issue. In general, I don't think we should ever use imports that directly reach into `<package>/src/`.

Possible fixes...
- `import { ... } from 'decap-cms-lib-util/dist/esm/types'`
- re-export these methods in `decap-cms-lib-util/src/index.ts` and `import { ... } from 'decap-cms-lib-util'`
- move these `isImmutable*` methods into `decap-cms-core`, as they aren't used anywhere else currently